py-polars/polars/series/series.py

```diff
@@ -608,16 +608,42 @@ def eq_missing(self, other: Expr) -> Expr:  # type: ignore[misc]
 
     def eq_missing(self, other: Any) -> Self | Expr:
         """
-        Method equivalent of equality operator ``expr == other`` where `None` == None`.
+        Method equivalent of equality operator ``series == other`` where `None` == None`.
 
-        This differs from default ``ne`` where null values are propagated.
+        This differs from the standard ``ne`` where null values are propagated.
 
         Parameters
         ----------
         other
             A literal or expression value to compare with.
 
-        """
+        Examples
+        --------
+        >>> s1 = pl.Series("a", [333, 200, None])
+        >>> s2 = pl.Series("a", [100, 200, None])
+        >>> s1.eq(s2)
+        shape: (3,)
+        Series: 'a' [bool]
+        [
+            false
+            true
+            null
+        ]
+        >>> s1.eq_missing(s2)
+        shape: (3,)
+        Series: 'a' [bool]
+        [
+            false
+            true
+            true
+        ]
+
+        See Also
+        --------
+        ne_missing
+        eq
```

Comment:
See Also section should be before Examples!

Reply:
Ok; we're definitely not doing that consistently though. 
Can do a run-through and update elsewhere too.

Reply:
Sure that would be welcome.

The guidelines are here:
https://numpydoc.readthedocs.io/en/latest/format.html#sections